documentation/reference/qfieldcloud/architecture.en.md

```diff
@@ -174,6 +177,9 @@ Stores data for the **[`minio`] S3 service**.
 
 Stores data for the **[`webdav`] storage service**.
 
+!!! info
+    The webdav storage is optional, it is not a requirement for the system to work properly.
+
```

Comment:
One warning above is probably enough?  You can ammend the previous sentence:

Stores data for the **[`webdav`] storage service** if present.